packages/ag-grid-community/src/valueService/cellApi.ts

```diff
@@ -14,14 +14,9 @@ export function getCellValue<TValue = any>(beans: BeanCollection, params: GetCel
     if (_missing(column)) {
         return null;
     }
-
-    const value = beans.valueSvc.getValueForDisplay(column, rowNode).value;
-
+    const result = beans.valueSvc?.getValueForDisplay(column, rowNode, useFormatter);
```

Comment:
```suggestion
    const result = beans.valueSvc.getValueForDisplay(column, rowNode, useFormatter);
```
Should always be defined